.github/CHANGELOG.md

```diff
@@ -510,6 +510,11 @@
 
 <h3>Breaking changes</h3>
 
+* A deprecation warning is now raised when loading content from the `qnn` module. In release 
+  `0.16.0`, the `qnn` module will no-longer be automatically loaded due to its dependency on
+  TensorFlow and Torch. Instead, users will need to additionally `import pennylane.qnn`.
```

Comment:
I think we'd rather recommend `from pennylane import qnn`, no? Otherwise they'll have to write `pennylane.qnn` every time

Reply:
If you do

```python
import pennylane as qml
import pennylane.qnn
```

then

```python
qml.qnn.KerasLayer
```

will continue to work

Reply:
It surprised me, but adding `import pennylane.qnn` allows you to keep using `qml.qnn.Keras` without changing:
```bash
>>> import pennylane as qml
>>> import pennylane.qnn
2021-03-26 10:33:30.243510: W tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2021-03-26 10:33:30.243542: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
>>> qml.qnn.KerasLayer
<class 'pennylane.qnn.keras.KerasLayer'>
```
The same also works with `from pennylane import qnn` as you suggest:
```bash
>>> import pennylane as qml
>>> from pennylane import qnn
2021-03-26 11:25:06.941368: W tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2021-03-26 11:25:06.941398: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
>>> qml.qnn.KerasLayer
<class 'pennylane.qnn.keras.KerasLayer'>
```

Reply:
Nice, good to know! 
It's more typing though :laughing: